### PR TITLE
[LIB-412] Nested models props

### PR DESCRIPTION
**Pull request by jakubbilko**

### CODE REVIEW

**Comment by jakubbilko** [comment]
@dancio review pls ;)

File: src/models/dataobject.js [hunk 1]
```diff
@@ -7,11 +7,11 @@ const DataObjectMeta = Meta({
   endpoints: {
     'detail': {
       'methods': ['delete', 'patch', 'put', 'get'],
-      'path': '/v1/instances/{instance}/classes/{class}/objects/{id}/'
+      'path': '/v1/instances/{instanceName}/classes/{class}/objects/{id}/'
```
Comment (dancio) [inline src/models/dataobject.js]: `className`?

File: src/models/profile.js [hunk 2]
```diff
@@ -7,11 +7,11 @@ const ProfileMeta = Meta({
   endpoints: {
     'detail': {
       'methods': ['delete', 'patch', 'put', 'get'],
-      'path': '/v1/instances/{instance}/classes/{class}/objects/{id}/'
+      'path': '/v1/instances/{instanceName}/classes/{class}/objects/{id}/'
```
Comment (dancio) [inline src/models/profile.js]: same here `className`?

File: src/models/webhooktrace.js [hunk 3]
```diff
@@ -7,11 +7,11 @@ const WebhookTraceMeta = Meta({
   endpoints: {
     'detail': {
       'methods': ['delete', 'patch', 'put', 'get'],
-      'path': '/v1/instances/{instance}/webhooks/{name}/traces/{traceId}/'
+      'path': '/v1/instances/{instanceName}/webhooks/{name}/traces/{traceId}/'
```
Comment (dancio) [inline src/models/webhooktrace.js]: `name` -> `webhookName`, `traceId` -> `id`?

File: src/models/codeboxtrace.js [hunk 4]
```diff
@@ -7,11 +7,11 @@ const CodeBoxTraceMeta = Meta({
   endpoints: {
     'detail': {
       'methods': ['delete', 'patch', 'put', 'get'],
-      'path': '/v1/instances/{instance}/codeboxes/{codeboxId}/traces/{traceId}/'
+      'path': '/v1/instances/{instanceName}/codeboxes/{codeboxId}/traces/{traceId}/'
```
Comment (dancio) [inline src/models/codeboxtrace.js]: `traceId` -> `id`?

**Comment by dancio** [comment]
let's go with this

**Pull request merged**